docs/source/index.rst

```diff
@@ -45,17 +45,17 @@ Features
 --------
 
 - Asynchronous
-- Be awesome
-- Make things faster
-- Have `FSM <https://en.wikipedia.org/wiki/Finite-state_machine>`_
-- Can reply into webhook
+- Awesome
+- Makes things faster
+- Has `FSM <https://en.wikipedia.org/wiki/Finite-state_machine>`_
+- Supports webhook
```

Comment:
It's wrong edit!
Right: Supports making requests when getting webhook updates

Reply:
Thanks a lot. Fixed.